ruby/app/models/Admin.rb

```diff
@@ -5,6 +5,13 @@ def self.get(public_token)
     # A start and end date are needed for a payroll report. The dates hard coded below will return a proper report from the sandbox environment
     report_id = Truv.requestPayrollReport(access_token, '2020-01-01', '2020-02-01')['payroll_report_id']
     payroll = Truv.getPayrollById(report_id)
+
+    if payroll["status"] != "done"
```

Comment:
Will we always succeed in two seconds with the singular retry?

Reply:
Well, it's a simple way. I guess it's enough for quickstart.